### DISCUSSION TITLE
GitHub Marketplace Renovate Runs Reduced?

### DISCUSSION THREAD

**Discussion by chezsmithy** [renovatebot/renovate#29217] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

We have observed through logs that the mend github marketplace offering seems to be performing less automated runs per day against our github repositories. 

It appeared to be running approximately 8x/day. A week ago that dropped to 4x/day. In the last couple days seems more like 2x/day.

For teams that have automerge turned on for their Github repos, this dramatically impacts how long a security PR will stay open. Previously, automerge would happen within an hour or two after the PR was posted. Now it can be open for 12 hours or more.

We are curious if this is a deliberate change or perhaps a result of a recent configuration change?

Thank you for investigating.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
There was a temporary scaling problem for workers which should now be resolved.

Can you use platform automerge? It means GitHub can automerge once tests pass and you don't need to wait for another run.

**Comment by alper** [comment]
In general are all these runs included in the free plan? I'm not sure whether I can expect a random bill to hit us or not.

**Reply by rarkins** [reply]
They are all included. There are no paid plan for the Renovate app on github.com, and definitely no possibility about getting a bill
